src/algebra/add_comm_torsor.lean

```diff
@@ -0,0 +1,184 @@
+/-
+Copyright (c) 2020 Joseph Myers. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: Joseph Myers.
+
+Torsors of additive commutative group actions, to the extent needed
+for affine spaces.
```

Comment:
We no longer use the copyright header for documentation.
```suggestion
```

Reply:
Change applied - https://leanprover-community.github.io/contribute/style.html needs updating, as that's where I took the header layout from.